recipes/gsoap/all/conanfile.py

```diff
@@ -8,8 +8,8 @@ class ConanFileDefault(ConanFile):
                   "REST XML Web services and generic C/C++ XML data bindings."
     topics = ("conan", "gsoap", "logging")
     url = "https://github.com/conan-io/conan-center-index"
-    homepage = "https://www.genivia.com"
-    license = "gSOAP-1.3b"
+    homepage = "https://sourceforge.net/projects/gsoap2"
+    license = "GPLv2"
```

Comment:
```suggestion
    license = ("gSOAP-1.3b", "GPL-2.0-or-later")
```


https://spdx.org/licenses/gSOAP-1.3b.html
https://spdx.org/licenses/GPL-2.0-or-later.html


According LICENSE file:

> The gSOAP 2.8 releases, including all 2.8.x updates, are distributed under:
>
> 1) The gSOAP Public License 1.3 (which is based on the Mozilla public license
>   1.1).
> 2) GPL v2 (GNU Public License, a common open-source software license) covers
>   all of the gSOAP software, see GPLv2_license.txt